packages/editor/src/components/projects/ProjectsPage.tsx

```diff
@@ -1,51 +1,194 @@
-import { Button, MediumButton } from '../inputs/Button'
-import { connectMenu, ContextMenu, MenuItem } from '../layout/ContextMenu'
-import {
-  ErrorMessage,
-  ProjectGrid,
-  ProjectGridContainer,
-  ProjectGridContent,
-  ProjectGridHeader,
-  ProjectGridHeaderRow
-} from './ProjectGrid'
-import templates from './templates'
 import React, { useEffect, useState, useRef } from 'react'
 import { useTranslation } from 'react-i18next'
-import { withRouter } from 'react-router-dom'
-import { StyledProjectsContainer, StyledProjectsSection, WelcomeContainer } from '../../pages/projectUtility'
+import { useHistory } from 'react-router-dom'
+import {
+  Accordion,
+  AccordionDetails,
+  AccordionSummary,
+  IconButton,
+  InputBase,
+  Menu,
+  Paper,
+  MenuItem
+} from '@mui/material'
+import { ArrowRightRounded, Clear, FilterList, Search, ManageAccounts, Check } from '@mui/icons-material'
 import { useAuthState } from '@xrengine/client-core/src/user/services/AuthService'
+import { useDispatch } from '@xrengine/client-core/src/store'
+import { ProjectService } from '@xrengine/client-core/src/common/services/ProjectService'
+import { ProjectInterface } from '@xrengine/common/src/interfaces/ProjectInterface'
+import { Button, MediumButton } from '../inputs/Button'
+import { connectMenu, ContextMenu } from '../layout/ContextMenu'
+import { ErrorMessage } from './ProjectGrid'
+import templates from './templates'
 import { getProjects } from '../../functions/projectFunctions'
 import { CreateProjectModal } from './CreateProjectModal'
-import { ProjectService } from '@xrengine/client-core/src/common/services/ProjectService'
-import { useDispatch } from '@xrengine/client-core/src/store'
 import { EditorAction } from '../../services/EditorServices'
-import { ProjectInterface } from '@xrengine/common/src/interfaces/ProjectInterface'
+import styles from './styles.module.scss'
 
 const contextMenuId = 'project-menu'
 
+const OfficialProjectData = [
+  {
+    id: '1570ae11-889a-11ec-886e-b126f7590685',
+    name: 'Maps',
+    repositoryPath: '',
+    storageProviderPath: 'https://localhost:8642/projects/default-project/',
```

Comment:
storage provider path here should be `''` and repository path should be https://github.com/XRFoundation/XREngine-Project-Maps

all other repos can be found https://github.com/XRFoundation/XREngine/issues/4905

Reply:
changed